src/event-handler/README.md

```diff
@@ -31,7 +31,7 @@ const { verify } = require('@octokit/webhooks')
 const secret = 'mysecret'
 
 if (!verify(secret, request.payload, request.headers['x-hub-signature'])) {
-  throw new Error('Signature does not match event payload & secret')
+  throw new Error('[@octokit/webhooks] Signature does not match event payload & secret')
```

Comment:
As this is user code, and not part of `@octokit/webhooks`, we don't need to add the prefix here. It's just a usage example. Can you remove it in this place, please?

Reply:
Done!